scripts/common.sh

```diff
@@ -40,6 +40,26 @@ else
     function printLog { echo "$(tput setaf 3)$1$(tput sgr0)"; }
 fi
 
+function checkDputEntries
+{
+    local pattern="$1"
+    grep "${pattern}" /etc/dput.cf --quiet || \
+        fail "Error: Missing ${pattern//\\/} section in /etc/dput.cf (check top comment in ${0} for more information)."
+}
+
+function sourcePPAConfig
+{
+    # source keyid and email from .release_ppa_auth
```

Comment:
I just noticed that this will accept variables exported from the shell - but only if `.release_ppa_auth` does not override them. Is that intentional? If not, maybe we should assert against that:

```suggestion
    [[ "$LAUNCHPAD_KEYID" == "" && "$LAUNCHPAD_EMAIL" == "" ]] || fail

    # source keyid and email from .release_ppa_auth
```

And if not, then it's actually not the way I'd expect. I think that normally env variables are used to override config settings and not the other way around.